src/openforms/js/components/admin/form_design/variables/prefill/PrefillSummary.js

```diff
@@ -0,0 +1,124 @@
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {normalizeErrors} from 'components/admin/forms/Field';
+import {EditIcon, ErrorIcon} from 'components/admin/icons';
+import {FormModal} from 'components/admin/modals';
+import ErrorBoundary from 'components/errors/ErrorBoundary';
+
+import {IDENTIFIER_ROLE_CHOICES} from '../constants';
+import PrefillConfigurationForm from './PrefillConfigurationForm';
+
+const PrefillSummary = ({
+  plugin = '',
+  attribute = '',
+  identifierRole = 'main',
+  onChange = undefined,
+  errors = {},
+}) => {
+  const intl = useIntl();
+  const [modalOpen, setModalOpen] = useState(false);
+  const identifierRoleMsg = IDENTIFIER_ROLE_CHOICES[identifierRole] || IDENTIFIER_ROLE_CHOICES.main;
+
+  const [hasPluginErrors, pluginErrors] = normalizeErrors(errors.prefillPlugin, intl);
+  const [hasAttributeErrors, attributeErrors] = normalizeErrors(errors.prefillAttribute, intl);
+  const [hasIdentifierRoleErrors, identifierRoleErrors] = normalizeErrors(
+    errors.prefillIdentifierRole,
+    intl
+  );
+
+  const hasErrors = hasPluginErrors || hasAttributeErrors || hasIdentifierRoleErrors;
+
+  const icons = (
+    <div style={{display: 'flex', gap: '6px', alignItems: 'center'}}>
+      {hasErrors && (
+        <span className="icon icon--danger icon--compact icon--no-pointer">
+          <ErrorIcon
+            text={intl.formatMessage({
+              description: 'Prefill configuration errors icon text',
+              defaultMessage: 'There are errors in the prefill configuration.',
+            })}
+          />
+        </span>
+      )}
+      {/* Modal control */}
+      {onChange && (
+        <EditIcon
+          label={intl.formatMessage({
+            defaultMessage: 'Edit prefill configuration',
+            description: "'Edit variable prefill' icon label",
+          })}
+          onClick={() => setModalOpen(!modalOpen)}
+        />
+      )}
+    </div>
+  );
+
+  return (
+    <div style={{display: 'flex', justifyContent: 'space-between', alignItems: 'center'}}>
+      {plugin === '' && attribute === '' ? (
+        '-'
+      ) : (
+        <FormattedMessage
+          tagName="span"
+          description="Prefill configuration summary"
+          defaultMessage="<code>{attr}</code> from <code>{plugin}</code> (of the {role})"
```

Comment:
indeed - in the preview we don't have the nice labels available for the values. The old situation also showed the raw value in its own column, so it's not making the situation _worse_.

On the other hand, Joeri was inclined to keep using the raw values because of familiarity with the attributes that already exists, so for now it's a wontfix, maybe even a wonteverfix.